### ISSUE TITLE
Fishlim hide mode

### ISSUE THREAD

**Issue by skydrome** [hexchat/hexchat#2493]
Can we make this an option to not show every line?
https://github.com/hexchat/hexchat/blob/71eb79fee4881a9ad06012a642b8deb3b4453c77/plugins/fishlim/plugin_hexchat.c#L240

**Comment by iyx** [comment]
Yes, I made this request [here](https://github.com/BakasuraRCE/hexchat-fishlim-reloaded/issues/4) but the original dev seems to be not around.

The use of encryption flag/mode would be much better if it was optional, or hidden.

**Comment by kayfab3** [comment]
It would be nice to have three alternate user configured options to default [ECB] and [CBC] prefixes

- set and display a custom prefix for CBC and ECB modes.  
- set and display a generic encrypted prefix instead of prefix for CBC and ECB modes.
- hide ECB and CBC prefix altogether instead making all encrypted text a specific color.

**Comment by kayfab3** [comment]
@BakasuraRCE could you take a look at this ticket.

**Comment by BakasuraRCE** [comment]
At this time I no longer have time to fix plugin failures, please support in the community or try to fix the code by itself. Thanks!
